examples/manifests/kustomization.yaml

```diff
@@ -11,8 +11,8 @@ replicas:
   name: cloud-consumer-deployment
 images:
 - name: cloud-event-consumer
-  newName: quay.io/redhat-cne/cloud-event-consumer
-  newTag: latest
+  newName: quay.io/aneeshkp/cloud-event-consumer
+  newTag: 4.13-sub
 - name: cloud-event-sidecar
-  newName: quay.io/redhat-cne/cloud-event-proxy
-  newTag: latest
+  newName: quay.io/aneeshkp/cloud-event-proxy
+  newTag: 4.13-sub3
```

Comment:
remove temporary changes.

Reply:
fixed . sorry,I forgot not to include this.